main.go

```diff
@@ -135,7 +135,17 @@ var timeoutFlag = cli.StringFlag{
 
 var pushFlag = cli.BoolFlag{
 	Name:  "push",
-	Usage: "Push  mode forces the daemon to start making beacon requests to the other node, instead of waiting the other nodes contact it to catch-up on the round",
+	Usage: "Push mode forces the daemon to start making beacon requests to the other node, instead of waiting the other nodes contact it to catch-up on the round",
+}
+
+var sourceFlag = cli.StringFlag{
+	Name:  "source",
+	Usage: "Source flag allows to provide an executable which output will be used as additional entropy during resharing step.",
+}
+
+var userEntropyOnlyFlag = cli.BoolFlag{
+	Name:  "userEntropyOnly",
```

Comment:
something like `source-only` or `user-source-onlŷ` or any other flag that is lowercase and separated. Sorry for the nitpick but that's kinda like a standard way to define flags so let's stick to what ppl know.